src/Tools/GetDocumentInsider/src/Commands/GetDocumentCommand.cs

```diff
@@ -32,6 +33,7 @@ public override void Configure(CommandLineApplication command)
         _output = command.Option("--output <Directory>", Resources.OutputDescription);
         _openApiVersion = command.Option("--openapi-version <Version>", Resources.OpenApiVersionDescription);
         _documentName = command.Option("--document-name <Name>", Resources.DocumentNameDescription);
+        _projectNameOverride = command.Option("--projectName-override <Name>", Resources.ProjectNameOverrideDescription);
```

Comment:
```suggestion
        _fileName = command.Option("--file-name-override <Name>", Resources.FileNameDescription);
```

I think `fileName` would be a clearer indication of what is happening here.